src/main/resources/anniv5Resources/localization/eng/bladestormpack/Powerstrings.json

```diff
@@ -0,0 +1,37 @@
+{
+  "${ModID}:EscalatingBreezePower": {
+    "NAME": "Escalating Breeze",
+    "DESCRIPTIONS": [
+      "Retain up to #b",
+      " card this turn. Chosen Attacks will cost #b",
+      " cards this turn. Chosen Attacks will cost #b",
+      " less next turn."
+    ]
+  },
+  "${ModID}:DownwindBlowPower": {
```

Comment:
I don't think this entry is used since Downwind Blow reuses Instant Death's `CloudPower`, so this entry can be deleted.